www/install/php/Update-20.04.0-beta.1.php

```diff
@@ -80,47 +78,79 @@
+                $versionOfTheUpgrade . "Unable to move engine's broker modules configuration from xml to json format"
+            );
+        };
     }
-} catch (\PDOException $e) {
-    $centreonLog->insertLog(
-        2,
-        "UPGRADE : 20.04.0-beta.1 Unable to move engine's broker modules configuration from xml to json format"
-    );
-    throw new \PDOException($e);
-}
 
+    /**
+     * Change broker sql output form
+     */
+    // set common error message on failure
+    $errorMessage = $versionOfTheUpgrade .
+        "Unable to move engine's broker modules configuration from xml to json format";
 
-// Change broker sql output form
-
-try {
     // reorganise existing input form
-    $pearDB->query(
-        "UPDATE cb_type_field_relation AS A INNER JOIN cb_type_field_relation AS B ON A.cb_type_id = B.cb_type_id
+    $query = "UPDATE cb_type_field_relation AS A INNER JOIN cb_type_field_relation AS B ON A.cb_type_id = B.cb_type_id
         SET A.`order_display` = 8 
-        WHERE B.`cb_field_id` = (SELECT f.cb_field_id FROM cb_field f WHERE f.fieldname = 'buffering_timeout')"
-    );
+        WHERE B.`cb_field_id` = (SELECT f.cb_field_id FROM cb_field f WHERE f.fieldname = 'buffering_timeout')";
+    if ($pearDB->query($query) === false) {
+        throw new \PDOException(
+            $errorMessage . " - While trying to update 'cb_type_field_relation' table data");
+    }
 
     // add new connections_count input
-    $pearDB->query(
-        "INSERT INTO `cb_field` (`fieldname`, `displayname`, `description`, `fieldtype`, `external`) 
-        VALUES ('connections_count', 'Number of connection to the database', 'Usually cpus/2', 'int', NULL)"
-    );
+    $query = "INSERT INTO `cb_field` (`fieldname`, `displayname`, `description`, `fieldtype`, `external`) 
+        VALUES ('connections_count', 'Number of connection to the database', 'Usually cpus/2', 'int', NULL)";
+    if ($pearDB->query($query) === false) {
+        throw new \PDOException(
+            $errorMessage . " - While trying to insert in 'cb_field' table new values"
+        );
+    }
 
     // add relation
-    $pearDB->query(
-        "INSERT INTO `cb_type_field_relation` (`cb_type_id`, `cb_field_id`, `is_required`, `order_display`, `jshook_name`, `jshook_arguments`)
+    $query = "INSERT INTO `cb_type_field_relation` (
+            `cb_type_id`,
+            `cb_field_id`,
+            `is_required`,
+            `order_display`,
+            `jshook_name`,
+            `jshook_arguments`
+        )
         VALUES (
             (SELECT `cb_type_id` FROM `cb_type` WHERE `type_shortname` = 'sql'),
             (SELECT `cb_field_id` FROM `cb_field` WHERE `fieldname` = 'connections_count'),
-            0, 7, 'countConnections', '{\"target\": \"connections_count\"}'
-        )"
-    );
+            0,
+            7,
+            'countConnections',
+            '{\"target\": \"connections_count\"}'
+        )";
+    if ($pearDB->query($query) === false) {
```

Comment:
The CentreonDB class is already configured to generate an exception if an error occurs.